base/shell/explorer/lang/tr-TR.rc

```diff
@@ -20,10 +21,10 @@ BEGIN
         MENUITEM "&Pencereleri basamakla", ID_SHELL_CMD_CASCADE_WND
         MENUITEM "P&encereleri yan yana göster", ID_SHELL_CMD_TILE_WND_H
         MENUITEM "Pe&ncereleri üst üste göster", ID_SHELL_CMD_TILE_WND_V
-        MENUITEM "&Masaüstünü göster", ID_SHELL_CMD_SHOW_DESKTOP
+        MENUITEM "&Masaüstünü/Pencereleri Göster", ID_SHELL_CMD_SHOW_DESKTOP
```

Comment:
```suggestion
        MENUITEM "&Masaüstünü Göster", ID_SHELL_CMD_SHOW_DESKTOP
```
This one had to stay as it is (it's one of the traybar context menu item; it shows by default "Show the desktop", but it gets dynamically changed to "Show Open Windows" in case you have previously showed the desktop).